### ISSUE TITLE
NullPointerException when super class returns null as qualified name

### ISSUE THREAD

**Issue by SirTakobi** [wt-io-it/odoo-pycharm-plugin#70]
Plugin Version: 0.6.5
IntelliJ Version: PyCharm 2022.2.3
IntelliJ Build: PC-222.4345.23

<details><summary>java.lang.NullPointerException</summary>

```
java.lang.NullPointerException
	at java.base/jdk.internal.reflect.GeneratedConstructorAccessor155.newInstance(Unknown Source)
	...
	at java.base/java.util.concurrent.ForkJoinTask.join(ForkJoinTask.java:672)
	at com.intellij.concurrency.JobLauncherImpl.processQueue(JobLauncherImpl.java:357)
	...
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:184)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	...
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.NullPointerException: Cannot invoke "String.split(String)" because "currentName" is null
	at at.wtioit.intellij.plugins.odoo.OdooModelPsiElementMatcherUtil.definedByImport(OdooModelPsiElementMatcherUtil.java:309)
	at at.wtioit.intellij.plugins.odoo.OdooModelPsiElementMatcherUtil.isOdooClassName(OdooModelPsiElementMatcherUtil.java:298)
	at at.wtioit.intellij.plugins.odoo.OdooModelPsiElementMatcherUtil.isOdooModelClassName(OdooModelPsiElementMatcherUtil.java:280)
	at at.wtioit.intellij.plugins.odoo.OdooModelPsiElementMatcherUtil.isOdooModelDefinition(OdooModelPsiElementMatcherUtil.java:256)
	at at.wtioit.intellij.plugins.odoo.OdooModelPsiElementMatcherUtil.isOdooModelPsiElement(OdooModelPsiElementMatcherUtil.java:132)
	at at.wtioit.intellij.plugins.odoo.models.inspection.MissingModelDefinitionInspection$1.visitElement(MissingModelDefinitionInspection.java:51)
	at com.intellij.psi.impl.source.tree.LeafPsiElement.accept(LeafPsiElement.java:208)
	...
	at com.intellij.concurrency.JobLauncherImpl$1MyProcessQueueTask.call(JobLauncherImpl.java:270)
	at java.base/java.util.concurrent.ForkJoinTask$AdaptedCallable.exec(ForkJoinTask.java:1428)
	... 5 more
```
</details>

**Comment by ap-wtioit** [comment]
@SirTakobi thanks for reporting this, will work on a fix tomorrow

TODOs:
* [ ] find test case
* [ ] add `@NotNull` for call tree for arguments that should not be null
```
                        String qualifiedName = superClass.getQualifiedName();
                        if (qualifiedName != null && isOdooModelClassName(qualifiedName, pyClass)) {
                            return true;
                        }
```

**Comment by bigbear3001** [comment]
this was fixed in 0.6.6 (pending jetbrains approval), if you are eager to test it you can use the jar file from https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.6

**Comment by bigbear3001** [comment]
0.6.6 is now available in the marketplace
